### PR TITLE
Create Letter-from-the-Board.md

### PR DESCRIPTION
**Pull request by MartinOtter**

### CODE REVIEW

**Comment by beutlich** [comment]
@filip-jezek The link to the https://github.com/modelica/MAP-LIB_ProjectRules repo will work out soon, when we switch it to a public repo.

@dietmarw

**Comment by beutlich** [comment]
@MartinOtter Should we make "The International Modelica Conference" more explicit as "The 13th International Modelica Conference"?

File: _2018-01/Letter-from-the-Board.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  - The [second Japanese Modelica Conference](https://www.modelica.org/events/modelica2018japan)
+    will take place **May 17-19, 2018** in Tokyo, Japan.
+
+  - The [first American Modelica Conference](https://www.modelica.org/events/modelica2018Americas)
+    will take place **Oct. 8-10, 2018** in Cambridge, MA, U.S.A.
+
+
+If you would like to organize a Modelica Conference, please inspect the Modelica Conference series
+[web page](https://www.modelica.org/publications/ModelicaConference).
```
Comment (dietmarw) [inline _2018-01/Letter-from-the-Board.md]: This page seems to be out of date. At least the last conference is not mentioned there. Have not checked the rest in detail. That should be fixed before the newsletter is updated.

**Comment by filip-jezek** [comment]
I am closing this pull request and create tasks instead.

File: _2018-01/Letter-from-the-Board.md (hunk 1 above)
Comment (MartinOtter) [reply _2018-01/Letter-from-the-Board.md]: The missing Modelica'2017 conference was added

**Pull request merged**